src/dashboard/cd-dashboard-stats.vue

```diff
@@ -87,9 +91,9 @@
         )).body.length;
       },
       async getDojoUsers() {
-        this.dojoUsers = (await DojoService.getDojoUsers(this.dojoId, { userType: 'attendee-u13', fields: ['gender'] })).body.response;
-        this.dojoUsers = this.dojoUsers.concat((await DojoService.getDojoUsers(this.dojoId, { userType: 'attendee-o13', fields: ['gender'] })).body.response);
-        this.genders = this.dojoUsers.reduce((red, user) => {
+        let dojoUsers = (await DojoService.getDojoUsers(this.dojoId, { userType: 'attendee-u13', fields: ['gender'] })).body.response;
```

Comment:
any reason for using let instead of const?

Reply:
It is reassigned the line after

Reply:
ah ok